CHANGELOG.md

```diff
@@ -2317,6 +2317,8 @@ v0.24.0 (2022-04-07)
 
 ### Features
 
+- Adding more option to enable required mongo stats to monitor. (@Nitesh-vaidyanath)
```

Comment:
This changelog entry is placed in a old release section. Please, move it to unreleased section.